pallets/ocex/src/lib.rs

```diff
@@ -649,6 +658,20 @@ pub mod pallet {
 			debug!("registered enclave at time =>{:?}", report.timestamp);
 			Ok(())
 		}
+
+		/// In order to register itself - enclave accountid must be whitelisted called by
+		/// Governance
```

Comment:
Please update to:
```rust
		/// In order to register itself - enclave account id must be whitelisted and called by
		/// Governance
```

Reply:
Done ✅

Reply:
@felixfaisal I don't see any changes about this one...

Reply:
This has been updated now, Sorry for the confusion